src/redux/actions/rbac-actions.js

```diff
@@ -0,0 +1,10 @@
+import { FETCH_RBAC_GROUPS } from '../action-types';
+import { getRbacGroups } from '../../helpers/rbac/rbac-helper';
+
+export const fetchRbacGroups = () => ({
+  type: FETCH_RBAC_GROUPS,
+  payload: getRbacGroups().then(({ data }) => [
```

Comment:
you don't need the square brackets and cloning the array. `.map` already returns new array.

Reply:
I missed this one - changed now.